lib/manageiq/rack_attack.rb

```diff
@@ -0,0 +1,54 @@
+require 'rack/attack'
+
+module ManageIQ
+  module RackAttack
+    def self.enable
+      if !Rails.env.test? && ENV.fetch("CYPRESS", false).to_s != "true"
```

Comment:
Minor, but ENV will always return strings, so if we default to a string, we don't need the to_s

```suggestion
      if !Rails.env.test? && ENV.fetch("CYPRESS", "false") != "true"
```

Reply:
haha, good points...